recruiter/parser/hackernews.py

```diff
@@ -1,5 +1,7 @@
-from base import *
+from recruiter.parser.base import *
 from recruiter.candidate import Candidate
+import six
```

Comment:
@pvwoods It seems like grouping all of the "import"s together at the top followed by the "from"s has been a general coding standard I've seen. Does that sound good to you?